content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -92,7 +85,7 @@ updates the workload's PodSpec to include the `overhead` as described in the Run
 the Pod will be rejected. In the given example, since only the RuntimeClass name is specified, the admission controller mutates the Pod
 to include an `overhead`.
 
-After the RuntimeClass admission controller, you can check the updated PodSpec:
+After the RuntimeClass admission controller, you can check the updated Pod spec:
```

Comment:
```suggestion
After the RuntimeClass admission controller has made modifications, you can check the updated
overhead value:
```

(I wonder, could we also show this with `kubectl ... --dry-run=server`?)

Reply:
I'm not sure if dry run works for this case.